CONTRIBUTING.md

```diff
@@ -53,7 +51,8 @@ To minimize code duplication and maximize flexibility, bootstrap clusters with a
 
 ### A new Machine can be created in a declarative way
 
-**A new Machine can be created in a declarative way, including Kubernetes version and container runtime version. It should also be able to specify provider-specific information such as OS image, instance type, disk configuration, etc., though this will not be portable.**
+A new Machine can be created in a declarative way, including Kubernetes version (i.e. the version of the Kubelet).
```

Comment:
`including Kubernetes version`
perhaps a indefinite / definite article is needed here?

Reply:
may be just say ... specifying versions of various components like the kubelet

Reply:
done.